src/coll/error.rs

```diff
@@ -320,17 +332,19 @@ impl BulkWriteException {
         // Parse out any write errors.
         let w_errs = if let Some(&Bson::Array(ref errors)) = result.get("writeErrors") {
             if errors.is_empty() {
-                return Err(Error::ResponseError(
-                    "Server indicates a write error, but none were found.".to_owned()));
+                return Err(Error::ResponseError("Server indicates a write error, but none were \
+                                                 found."
+                    .to_owned()));
```

Comment:
actually weird formatting here, especially since the original seemed to be within the character limit.